lib/github_importer.rb

```diff
@@ -12,6 +12,8 @@ def initialize(oauth_token, repo)
   # @yield [HarvestedResource] Gives the harvested resource
   def import(harvest, pipeline)
     gh.contents(repo, path: pipeline.config.directory).each do |resource|
+      next if pipeline.filter && resource.name !~ pipeline.filter
```

Comment:
I don't understand how this is resources that match. It looks to me like we skip importing (`next`) if the file matches the filter. Right?